prov/shm/src/smr_progress.c

```diff
@@ -875,8 +875,12 @@ static int smr_progress_cmd_msg(struct smr_ep *ep, struct smr_cmd *cmd)
 		ofi_ep_lock_release(&ep->util_ep);
 		if (ret == -FI_ENOENT) {
 			ret = smr_alloc_cmd_ctx(ep, rx_entry, cmd);
-			if (ret)
+			if (ret) {
```

Comment:
there is one more places that this could fail, the `queue_msg` path

Reply:
added